src/org/labkey/test/tests/AttachmentFieldTest.java

```diff
@@ -114,13 +115,13 @@ public void testAttachmentFieldInLists()
         clickAndWait(Locator.linkWithText(listName));
         DataRegionTable listTable = new DataRegionTable("query", getDriver());
         listTable.clickInsertNewRow();
-        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_PDF);
+        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_FILE);
         clickButton("Submit");
 
         log("Verify file opened in browser");
-        Locator.linkWithText(SAMPLE_PDF.getName()).findElement(getDriver()).click();
+        Locator.tagWithAttributeContaining("img", "title", SAMPLE_FILE.getName()).findElement(getDriver()).click();
         switchToWindow(1);
-        checker().verifyTrue("Incorrect PDF file displayed", getCurrentRelativeURL().contains(SAMPLE_PDF.getName()));
+        Assertions.assertThat(getDriver().getCurrentUrl()).as("Incorrect file displayed").contains(SAMPLE_FILE.getName());
```

Comment:
Still failing on TeamCity. Need to wait for new window to load.
```suggestion
        waitFor(() -> getDriver().getCurrentUrl().startsWith("http"), "Tab failed to load", 5_000);
        Assertions.assertThat(getDriver().getCurrentUrl()).as("Incorrect file displayed").contains(SAMPLE_FILE.getName());
```